zipkin-lens/src/components/TracePage/SpanDetail/SpanTags.jsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withStyles } from '@material-ui/styles';
+import Box from '@material-ui/core/Box';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import grey from '@material-ui/core/colors/grey';
+
+import { spanTagsPropTypes } from '../../../prop-types';
+
+const style = {
+  cell: {
+    paddingTop: '8px',
+    paddingBottom: '8px',
+  },
+  key: {
+    color: grey[500],
+    fontWeight: 'bold',
+  },
+  value: {
+    fontSize: '1.05rem',
```

Comment:
Is this a fractional value?

In general I think using px is better than rem since it's difficult to know what's the final value in case of rem.

The reason some prefer rem is "to make it scale with users' setting", however, in practice, users are most likely to zoom their browser (e.g. `cmd +`) instead of going to settings and set a different font size. In these cases, px works much better